tools/taco.cpp

```diff
@@ -497,6 +497,24 @@ static bool setSchedulingCommands(vector<vector<string>> scheduleCommands, parse
 
       stmt = stmt.reorder(reorderedVars);
 
+    } else if (command == "mergeby") {
+      taco_uassert(scheduleCommand.size() == 2) << "'mergeby' scheduling directive takes 2 parameters: mergeby(i, strategy)";
```

Comment:
Since you've added the scheduling primitive to the CLI, add a CLI test as well that exercises it.